libr/debug/p/native/linux/linux_debug.c

```diff
@@ -97,6 +97,22 @@ int linux_handle_signals(RDebug *dbg, int tid) {
 		// siginfo.si_code -> HWBKPT, USER, KERNEL or WHAT
 		// TODO: DO MORE RDEBUGREASON HERE
 		switch (dbg->reason.signum) {
+		case SIGABRT: // 6 / SIGIOT // SIGABRT
+			dbg->reason.type = R_DEBUG_REASON_ABORT;
+			break;
+		case SIGALRM:
+			dbg->reason.type = R_DEBUG_REASON_SIGALRM;
+			alarm(0);
```

Comment:
why

Reply:
I believed I had to handle alarm syscall on the debugger like in an emulator. I doubt.

Reply:
Alarm doesnt handle the signal. And you want to handlw the signal in the debugee not the debugger. So this line isnt needed here